### DISCUSSION TITLE
How do I make Sidebar use the full height of the container?

### DISCUSSION THREAD

**Discussion by reubenfirmin** [themesberg/flowbite-svelte#1018] [Q&A]
I would like my sidebar to be full height, and for some reason it's resisting doing so - it sizes itself to content only. Is there a trick? Currently I have:

```
<Sidebar activeUrl={location.pathname} class="h-full" activeClass={activeClass}>
```

Obviously I can stack a filler div in the same column, but that's a bit messy.

**Comment by shinokada** [comment]
Have you tried `h-screen`?

**Reply by reubenfirmin** [reply]
My bad, it was a parent container constraining it. Thanks - h-screen does work.
